manifests/09-operator-cr.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: operator.openshift.io/v1
+kind: InsightsOperator
+metadata:
+  name: cluster
```

Comment:
You need to annotate this with `include.release.openshift.io/...` for the cluster profiles it should belong to.  See your other manifests for examples.  [As it stands now][1]:

```console
$ curl -s https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_insights-operator/666/pull-ci-openshift-insights-operator-master-e2e/1562059761355067392/artifacts/e2e/gather-extra/artifacts/pods/openshift-cluster-version_cluster-version-operator-5bff5b8bd6-2d6n7_cluster-version-operator.log | grep kind=InsightsOperator
I0823 13:42:20.564370       1 payload.go:210] excluding 0000_50_insights-operator_09-operator-cr.yaml group=operator.openshift.io kind=InsightsOperator namespace= name=cluster: no annotations
I0823 13:42:21.332159       1 payload.go:210] excluding 0000_50_insights-operator_09-operator-cr.yaml group=operator.openshift.io kind=InsightsOperator namespace= name=cluster: no annotations
```

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_insights-operator/666/pull-ci-openshift-insights-operator-master-e2e/1562059761355067392

Reply:
Thank you